policy/modules/system/userdomain.if

```diff
@@ -69,6 +69,9 @@ template(`userdom_base_user_template',`
 	dontaudit $1_t user_tty_device_t:chr_file ioctl;
 
 	kernel_read_kernel_sysctls($1_t)
+	kernel_read_crypto_sysctls($1_t)
+	kernel_read_fs_sysctls($1_t)
+	kernel_read_vm_overcommit_sysctl($1_t)
```

Comment:
What are these for?

Reply:
type=AVC msg=audit(1695241764.653:6170): avc:  granted  { read open } for  pid=285781 comm="http" path="/proc/sys/crypto/fips_enabled" dev="proc" ino=21350 scontext=etbe:user_r:user_t:s0-s0:c0.c1023 tcontext=system_u:object_r:sysctl_crypto_t:s0 tclass=file

The crypto sysctls are needed by some http/https programs among other things, seems like a lot of library code checks this.

Reply:
type=AVC msg=audit(1695242496.734:181): avc:  granted  { read open } for  pid=1932 comm=66757365206D61696E6C6F6F70 path="/proc/sys/fs/pipe-max-size" dev="proc" ino=27353 scontext=user_u:user_r:user_t:s0 tcontext=system_u:object_r:sysctl_fs_t:s0 tclass=file

This process is xdg-document-portal. Lots of programs check pipe-max-size.

type=AVC msg=audit(1695242498.119:183): avc:  granted  { read open } for  pid=1673 comm=5468726561642028706F6F6C656429 path="/proc/sys/vm/overcommit_memory" dev="proc" ino=29888 scontext=user_u:user_r:user_t:s0 tcontext=system_u:object_r:sysctl_vm_overcommit_t:s0 tclass=file

This process is KDE plasmashell. Lots of programs do checks like this.

I don't think there's a downside to allowing these to be read.

Reply:
Seems fine. Was just curious as I don't think I have seen these AVCs before.